### PR TITLE
Authors

### PR DESCRIPTION
**Pull request by JessSaavedra**
## Summary

- Changed articles and comments so that only their author can edit or delete them

## Trello Card

https://trello.com/c/bCeKXTOD

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ def edit
 
   def create
     @article = Article.new(article_params)
+    @article.user_id = current_user.id
```
Comment (epintos) [inline app/controllers/articles_controller.rb]: You can put @article.user = current_user

File: app/controllers/articles_controller.rb [hunk 2]
```diff
@@ -26,19 +27,32 @@ def show
   end
 
   def update
-    if @article.update(article_params)
-      redirect_to @article
+    if current_user.id == @article.user_id
```
Comment (epintos) [inline app/controllers/articles_controller.rb]: SAme here you can compare without using the id. Leave it now because you should change it with ppundit

**Comment by epintos** [comment]
:+1:

**Comment by JessSaavedra** [comment]
Merged manually

**Pull request closed without merging**